### PR TITLE
[fix bug 1208367] Autocomplete form for adding multiple curators.

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

**Comment by johngian** [comment]
Please rebase. Also tests are failing.

File: mozillians/groups/helpers.py [hunk 1]
```diff
@@ -13,6 +13,14 @@ def stringify_groups(groups):
     return u','.join([group.name for group in groups])
 
 
+def strignify_users(users):
```
Comment (johngian) [inline mozillians/groups/helpers.py]: Please fix typo here.

Comment (johngian) [inline mozillians/groups/helpers.py]: This also has other occurrences.

File: mozillians/users/models.py [hunk 2]
```diff
@@ -260,6 +260,14 @@ def _filter_accounts_privacy(self, accounts):
             return accounts.filter(privacy__gte=self._privacy_level)
         return accounts
 
+    @classmethod
+    def search_vouched(cls, query):
+        query = query.lower()
+        results = cls.objects.filter(full_name__icontains=query,
+                                     is_vouched=True)
```
Comment (johngian) [inline mozillians/users/models.py]: Please squash this in one line.

File: mozillians/users/models.py [hunk 3]
```diff
@@ -260,6 +260,14 @@ def _filter_accounts_privacy(self, accounts):
...
+        results = results.distinct()
+        return results
```
Comment (johngian) [inline mozillians/users/models.py]: You could instead skip the previous line and ``return results.distinct()``

**Comment by akatsoulas** [comment]
@johngian pr updated

**Comment by johngian** [comment]
Closing this one after #1334

**Pull request closed without merging**